sonic_platform_base/sonic_thermal_control/thermal_policy.py

```diff
@@ -0,0 +1,67 @@
+from .thermal_json_object import ThermalJsonObject
+
+
+class ThermalPolicy(object):
+    """
+    Class representing a thermal policy. A thermal policy object is initialized by policy.json.
```

Comment:
There are multiple comments in this PR which refer to the file "policy.json". However, this API allows the user to specify the path to the file in the `load()` function, thus the filename can differ. Consider changing these comments from "policy.json" to "JSON policy file".